packages/capabilities/src/space.js

```diff
@@ -65,13 +65,54 @@ export const allocate = capability({
 })
 
 /**
- * The capability grants permission for all content serve operations that fall under the "space/content/serve" namespace.
- * It can be derived from any of the `space/*` capability that has matching `with`.
+ * "Manage the serving of content owned by the subject Space."
+ *
+ * A Principal who may `space/content/serve/*` is permitted to perform all
+ * operations related to serving content owned by the Space, including actually
+ * serving it and recording egress charges.
  */
-
 export const contentServe = capability({
   can: 'space/content/serve/*',
+  /**
+   * The Space which contains the content. This Space will be charged egress
+   * fees if content is actually retrieved by way of this invocation.
+   */
   with: SpaceDID,
+  nb: Schema.struct({
+    /** The authorization token, if any, used for this request. */
+    token: Schema.string().nullable(),
+  }),
+  derives: equalWith,
+})
+
+/**
+ * "Serve content owned by the subject Space over HTTP."
+ *
+ * A Principal who may `space/content/serve/transport/http` is permitted to
+ * serve any content owned by the Space, in the manner of an [IPFS Gateway]. The
+ * content may be a Blob stored by a Storage Node, or indexed content stored
+ * within such Blobs (ie, Shards).
+ *
+ * Note that the args do not currently specify *what* content should be served.
+ * Invoking this command does not currently *serve* the content in any way, but
+ * merely validates the authority to do so. Currently, the entirety of a Space
+ * must use the same authorization, thus the content does not need to be
+ * identified. In the future, this command may refer directly to a piece of
+ * content by CID.
+ *
+ * [IPFS Gateway]: https://specs.ipfs.tech/http-gateways/path-gateway/
+ */
+export const transportHttp = capability({
+  can: 'space/content/serve/transport/http',
```

Comment:
IDK if "transport" is useful, makes it very verbose.

Suggestion: `space/content/serve/http`

Reply:
I agree. I'm not sure now who originally proposed `transport` and why, but I don't _think_ there was strong reasoning that went with it as much as putting something down.